daemon/userconsent_test.go

```diff
@@ -0,0 +1,323 @@
+//go:build moose
```

Comment:
As per previous comment, this can be removed. If this is here, this will only be executed with `cgo` tests for now without a need to do so.

Reply:
Done